CHANGELOG.md

```diff
@@ -65,7 +61,11 @@ See new schema (2.0) documentation [here](https://github.com/CDRH/datura/docs/sc
 
 ### Migration
 - check to make sure "text" xpath is doing desired behavior
-- use Elasticsearch 8.5 or higher and add authentication as described above if security is enabled. See [dev docs instructions](https://github.com/CDRH/cdrh_dev_docs/blob/update_elasticsearch_documentation/publishing/2_basic_requirements.md#downloading-elasticsearch).
+- use Elasticsearch 8.5 or higher and add authentication as described above if security is enabled. Add the following to `public.yml` or `private.yml` in the data repo:
```

Comment:
Please revise the text here to reflect that one doesn't need to look above for the auth info now.